content/en/events/2022/kcsna/social.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Contributor Social
+type: docs
+weight: 5
+---
+
+# Contributor Social
+
```

Comment:
The header is coming twice. Once because of the theme and then because of this header. Can we remove the header? I think what the theme is showing suffices.

![image](https://user-images.githubusercontent.com/10010419/182917244-77283562-f02d-47a5-ba00-c45ffaeae155.png)